app/src/main/java/org/jellyfin/mobile/player/source/ExoPlayerTrack.kt

```diff
@@ -9,6 +10,10 @@ sealed class ExoPlayerTrack(track: JSONObject) {
     val title: String = track.optString("DisplayTitle", "")
 
     class Video(track: JSONObject) : ExoPlayerTrack(track) {
+        val width: Int = track.optInt("Width", -1)
```

Comment:
We should move those strings to constants

Reply:
I don't know, we'll rip this out anyways one we get to the stage where we handle video playback with the new apiclient..